roles/common/tasks/host_validations.yml

```diff
@@ -79,7 +79,18 @@
   - validate
   - validate_tmp_access
   
-  - name: Validate enough free disk space for package installation
+- name: Validate enough free disk space for package installation
+  assert:
+    that: item.size_available > {{ required_disk_space_mb }}|float
+    fail_msg: >- 
+      "Not enough free disk space for package installation.  Minimum space required is {{ required_disk_space_mb }}MB, please free up more disk space 
+      to continue installation. To skip host validations, set validate_hosts to false."
+  with_items: "{{ ansible_mounts }}"
```

Comment:
Thanks!
Do we need to strictly validate for all mounts ? I feel there can be some mount points with less space which may not be involved in CP install at all. I feel, we should check if atleast one mount is above the requirement. Let me know what think

Reply:
The problem here is that we don't know what all the mounts are in advance.  We would have to ask the user to specify or provide some other input of specific mounts to check.  My recommendation would be to add a filter whereby we can exclude certain mounts.  As in we check everything by default but the customer can specify mount points to skip.

Reply:
I'm also not sure this is the correct PR for this discussion as this is to fix a formatting issue, would recommend filing a JIRA if you have an issue with the design/enhancement ideas.